core/scripts/PrecisionErrors.js

```diff
@@ -544,52 +560,54 @@ async function runExport() {
    * @notice PRE-TEST INVARIANTS
    */
   breakdown = {};
-  actualFeeMultiplier = await emp.cumulativeFeeMultiplier();
-  startingContractCollateral = await collateral.balanceOf(emp.address);
-  startingAdjustedContractCollateral = await emp.totalPositionCollateral();
   startingStoreCollateral = await collateral.balanceOf(store.address);
-  startingRawContractCollateral = await emp.rawTotalPositionCollateral();
+  actualFeeMultiplier = await emp.cumulativeFeeMultiplier();
+  contractCollateral = await collateral.balanceOf(emp.address);
+  adjustedCollateral = await emp.totalPositionCollateral();
+  rawContractCollateral = await emp.rawTotalPositionCollateral();
 
   // Test 1) Fee multiplier is set correctly.
   assert.equal(actualFeeMultiplier.toString(), testConfig.expectedFeeMultiplier);
 
   // Test 2) The collateral net-of-fees and collateral in contract should be equal to start.
-  assert.equal(startingContractCollateral.toString(), startingAdjustedContractCollateral.toString());
+  assert.equal(contractCollateral.toString(), adjustedCollateral.toString());
 
   // Log results in a table.
-  breakdown.expected = new CollateralBreakdown(startingContractCollateral);
-  breakdown.credited = new CollateralBreakdown(startingAdjustedContractCollateral);
-  breakdown.raw = new CollateralBreakdown(startingRawContractCollateral);
-  breakdown.feeMultiplier = new CollateralBreakdown(actualFeeMultiplier);
+  breakdown["Collateral owned by Contract"] = new CollateralBreakdown(contractCollateral);
+  breakdown["Collateral credited to Sponsors"] = new CollateralBreakdown(adjustedCollateral);
+  breakdown["Contract raw collateral"] = new CollateralBreakdown(rawContractCollateral);
+  breakdown["Cumulative Fee Multiplier"] = new CollateralBreakdown(actualFeeMultiplier);
   console.group("** Pre-Withdrawal: Actual and Credited Collateral should be equal **");
   console.table(breakdown);
   console.groupEnd();
 
   /**
-   * @notice RUN THE TEST ONCE
+   * @notice RUN THE TEST TEN TIMES
    */
-  // Note: Must call `requestWithdrawal()` instead of `withdraw()` because we are the only position. I didn't create another
-  // less-collateralized position because it would modify the total collateral and therefore the fee multiplier.
-  await emp.requestWithdrawal({ rawValue: testConfig.amountToWithdraw }, { from: sponsor });
-  // Move time forward. Need to set fees to 0 so as not to change the fee multiplier.
+  // Need to set fees to 0 so as not to change the fee multiplier.
   await store.setFixedOracleFeePerSecond({ rawValue: toWei("0") }, { from: contractDeployer });
-  startTime = await emp.getCurrentTime();
-  // Advance time to 1 second past the withdrawal liveness.
-  await emp.setCurrentTime(startTime.addn(1));
-  // Execute withdrawal request.
-  await emp.withdrawPassedRequest({ from: sponsor });
-
+  for (let i = 0; i < 10; i++) {
```

Comment:
This is a key change: As mentioned [here](https://docs.google.com/document/d/1p2-r8JoR-Unp8u3skPiCkGnKte0ImfrpXK2sXtrsPus), withdrawals remove *more* collateral from the contract than gets decremented from `rawCollateral`. Every withdraw induces 1e-18 of loss. However, in order to display the collateral credited to sponsors you'd call `getCollateral(sponsor)`. Recall that `getCollateral` computes (`rawCollateral * feeMultiplier`), which floors the result and cancels out the loss. Therefore, I run the withdraw 10 times in order to show the loss via `getCollateral()`